src/static/riot/competitions/detail/_header.tag

```diff
@@ -182,8 +184,9 @@
         self.create_dump = () => {
             CODALAB.api.create_dump(self.competition.id)
                 .done(data => {
-                    $(self.refs.dump_modal).modal('show')
-                    setTimeout(self.update_files, 2000)
+                    toastr.success("Success! Your competition dump is being created.")
+                    //$(self.refs.dump_modal).modal('show')
```

Comment:
can delete this, rather than comment it out.

Reply:
got it and on it.